Users/Tutorial.rst

```diff
@@ -148,16 +148,20 @@ This should yield something like this:
 
 ::
 
-    [Default]
+    [cli]
     bears = SpaceConsistencyBear
     files = src/*.c
-    use_spaces = yeah
+    use_spaces = True
 
 If you now invoke ``coala`` it will parse this ``.coafile`` from your
 current directory. This makes it easy to specify once for your project
 what is checked with which bears and make it available to all
 contributors.
 
+.. note::
+
+    The implicit inheritance feature of the Default section is now deprecated.
+
```

Comment:
I think instead of here placing a note while nothing in the doc has told the reader something about setting-inheritance yet, the `Setting Inheritance` paragraph should be improved instead :+1:

Reply:
Though hmm... at least I wouldn't do this comment here :+1: I think the issue isn't actually a newcomer issue because some knowledge about coala is required. So up to you :)

Reply:
@Makman2 since I am an absolute beginner, what should I then do about it?

Reply:
Just remove the note here :) We'll file another issue to be more precise on the section-inheritance feature :)
(sorry for taking so long :v:)

Reply:
@Makman2 I have made the said changes!
Sorry for responding so late!